packages/host/tests/acceptance/prerender-test.gts

```diff
@@ -158,6 +158,10 @@ module('Acceptance | prerender', function (hooks) {
     });
   });
 
+  hooks.afterEach(function () {
+    delete (globalThis as any).__lazilyLoadLinks;
```

Comment:
Not related to this PR, but if we don't delete __lazilyLoadLinks, the subsequent tests could fail because they expect it not to exist or false.